connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -93,9 +94,15 @@ object KustoWriter {
       KDSU.logInfo(myName, s"Successfully created temporary table $tmpTableName, will be deleted after completing the operation")
 
       kustoClient.setMappingOnStagingTableIfNeeded(stagingTableIngestionProperties, tableCoordinates.database, tmpTableName, table, crp)
-      if (stagingTableIngestionProperties.flushImmediately) {
-        KDSU.logWarn(myName, "It's not recommended to set flushImmediately to true")
+//    TODO remove until batching policy problem is good
```

Comment:
Why? Isn't `flushImmediately` still useful?

Reply:
It is 
I removed the warning as it seems that as long as we don't have   a solution with batching policy it is a good practice to use it